From fb5f05b0b395bdfee1808b85f673fcf901afca3f Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Thu, 24 Jan 2013 08:16:46 -0600
Subject: [PATCH 2/3] Change mailing list to vdpau@lists.freedesktop.org

Users should address questions there instead of to xorg@lists.freedesktop.org.

Signed-off-by: Aaron Plattner <aplattner@nvidia.com>
Acked-by: Robert Morell <rmorell@nvidia.com>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0cee3e5..03131f5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 AC_PREREQ(2.60)
 
-AC_INIT(libvdpau, 0.5, [xorg@lists.freedesktop.org], libvdpau)
+AC_INIT(libvdpau, 0.5, [vdpau@lists.freedesktop.org], libvdpau)
 AM_INIT_AUTOMAKE([dist-bzip2 foreign])
 AM_MAINTAINER_MODE
 
-- 
1.7.10.4

